@here/harp-mapview/lib/ThemeHelpers.ts

```diff
@@ -6,68 +6,103 @@
 
 import * as THREE from "three";
 
-import { Light, PixelFormat, TextureDataType, WrappingMode } from "@here/harp-datasource-protocol";
+import {
+    Light,
+    MagFilter,
+    MinFilter,
+    PixelFormat,
+    TextureDataType,
+    WrappingMode
+} from "@here/harp-datasource-protocol";
 
 /**
  * Returns `three.js` pixel format object basing on a [[PixelFormat]] specified.
  */
 export function toPixelFormat(format: PixelFormat): THREE.PixelFormat {
-    if (format === "Alpha") {
-        return THREE.AlphaFormat;
-    } else if (format === "RGB") {
-        return THREE.RGBFormat;
-    } else if (format === "RGBA") {
-        return THREE.RGBAFormat;
-    } else if (format === "Luminance") {
-        return THREE.LuminanceFormat;
-    } else if (format === "LuminanceAlpha") {
-        return THREE.LuminanceAlphaFormat;
-    } else if (format === "RGBE") {
-        return THREE.RGBEFormat;
-    } else if (format === "Depth") {
-        return THREE.DepthFormat;
-    } else if (format === "DepthStencil") {
-        return THREE.DepthStencilFormat;
+    switch (format) {
```

Comment:
Cool. I was doing C++ for too long. I did not even know that you can do a switch with strings.

Can you please add RedFormat as well. When I implemented this RedFormat was not in the type definitions yet.

Reply:
Will take care of it.